src/Core/Scheduler/Providers/GpxUtils.js

```diff
@@ -5,8 +5,12 @@
  */
 
 import * as THREE from 'three';
+import Line from 'three.meshline';
 import Fetcher from './Fetcher';
 import Coordinates from '../../Geographic/Coordinates';
+import Capabilities from '../../System/Capabilities';
+// eslint-disable-next-line
+import $3dTiles_Provider, { patchMaterial } from './3dTiles_Provider';
```

Comment:
I have to write $3dTiles_Provider to import patchMaterial otherwise there's a error